c-api/include/lol_html.h

```diff
@@ -685,7 +685,8 @@ void *lol_html_element_user_data_get(const lol_html_element_t *element);
 
 // Adds content handlers to the builder for the end tag of the given element.
 //
-// Subsequent calls to the method on the same element replace the previous handler.
+// Subsequent calls to the method on the same element adds new handler.
+// They will run in the order in which they were registered.
```

Comment:
```
// Returns 0 in case of success and -1 otherwise. The actual error message
// can be obtained using `lol_html_take_last_error` function.
```
The second sentence doesn't appear to be true anymore -- `lol_html_take_last_error()` now returns null.